src/runtime_src/driver/common/config_reader.cpp

```diff
@@ -54,27 +54,52 @@ get_self_path()
 #endif
 }
 
+/*
+ * Look for xrt.ini and i not found look for legacy sdaccel.ini
+ */
 static std::string
-get_ini_path()
+verify_ini_path(const boost::filesystem::path& dir)
 {
   try {
```

Comment:
Leave try/catch to calling function, which already is in try/catch block.

Reply:
Done